aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -29,8 +29,8 @@ Don’t collect feedback in the following use cases:
 
 | Label  | Region       | Purpose                                                              |                  Required                   | Notes                                                                                                      |
 | ------ | ------------ | -------------------------------------------------------------------- | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------- |
-| **1**  | **Title**    | Title prompting user for their feedback.            | <Checkmark a11yTitle="true" size="small" /> | "We'd love your feedback" is a nice, appreciative default title.       |
-| **2**  | **Body**     | The body may be populated as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> |  Question types can include StarRating, ThumbsRating, select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types. |                                                             
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
```

Comment:
one thing I wasnt sure about was the `required` should we put the checkmark even though its all required?

Reply:
I'm not following the question.

Reply:
if its all required do we still need this section for the table? I think of that as being overkill but my thought

Reply:
Gotcha. Good to consider. I think we are okay here -- it'll be consistent with other anatomy tables.